pycryptobot.py

```diff
@@ -1674,6 +1679,7 @@ def executeJob(
 
 def main():
     try:
+        _websocket = None
```

Comment:
I this to fix an existing bug where the WebSocket is used before being assigned?

Reply:
Yes it is